projects/js-packages/number-formatters/eslint.config.mjs

```diff
@@ -0,0 +1,27 @@
+// @ts-ignore
+import makeBaseConfig from 'jetpack-js-tools/eslintrc/base.mjs';
+
+export default [
+	...makeBaseConfig( import.meta.url ),
+	{
+		languageOptions: {
+			parserOptions: {
+				project: './tsconfig.json',
+			},
+		},
+		files: [ '**/*.ts', '**/*.tsx' ],
+		plugins: {
+			'@typescript-eslint': true,
+			jsdoc: true,
+		},
```

Comment:
This won't work with eslint 9, and I think it's redundant to what our base config already provides.